packages/amplify_auth_cognito/example/integration_test/utils/setup_utils.dart

```diff
@@ -17,10 +17,12 @@ import 'package:amplify_auth_cognito/amplify_auth_cognito.dart';
 import 'package:amplify_auth_cognito_example/amplifyconfiguration.dart';
 import 'package:amplify_flutter/amplify_flutter.dart';
 
-Future<void> configureAuth() async {
+Future<void> configureAuth({
+  List<AmplifyPluginInterface> additionalConfigs = const [],
```

Comment:
*nit:* I think `additionalConfigs` is what we have in the authenticator tests, but `additionalPlugins` probably makes more sense.

Reply:
Yeah you're absolutely correct.